### ISSUE TITLE
train accuracy exists problem.

### ISSUE THREAD

**Issue by starsun-z** [microsoft/VideoX#80]
There is a problem of train accuracy. The accuracy always  remains unchanged. I used UCF-101 for zero-shot and 10 classes for test in val.txt. So I want to know how you set it and how to set the super parameters. 

THANKS.

**Comment by nbl97** [comment]
The detailed hyperparameters of all settings are listed in the supplementary material (Tab. 1). For zero-shot test on UCF-101, we first train the model on Kinetics-400 to learn to capture temporal information and then test the model on UCF-101. We use the official test files that can be downloaded easily from the official website. If your custom test set contains only 10 classes, you may need to tune the hyperparameters slightly.
